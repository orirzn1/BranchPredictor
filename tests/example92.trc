16 2 11 1 global_history global_tables not_using_share
0xd374 N 0xf890
0xe24f7898 T 0x8e39c068
0xa40 N 0x57899e8
0xe24f7898 T 0x8e39c068
0xb7e460 T 0x6f63898
0xa40 N 0x57899e8
0xff14994 N 0xd1f74
0x2d0 T 0x0618
0x95666124 N 0x2e588c
0xf37f8 T 0xa89f434
0x95666124 T 0x2e588c
0x95666124 N 0x2e588c
0x2d0 N 0x0618
0xb7e460 N 0x6f63898
0x95666124 N 0x2e588c
0x95666124 N 0x2e588c
0xd374 T 0xf890
0xb7e460 N 0x6f63898
0xa40 T 0x57899e8
0xf37f8 N 0xa89f434
0x95666124 N 0x2e588c
0xb7e460 T 0x6f63898
0x53816c N 0x8779590
0x9eaf0 T 0x783d6b4
0xf37f8 T 0xa89f434
0xb7e460 T 0x6f63898
0x95666124 T 0x2e588c
0xff14994 T 0xd1f74
0xd374 N 0xf890
0x2d0 N 0x0618
0xf37f8 T 0xa89f434
0x95666124 N 0x2e588c
0xf37f8 T 0xa89f434
0xa40 N 0x57899e8
0xf37f8 N 0xa89f434
0x95666124 T 0x2e588c
0xd374 T 0xf890
0x9eaf0 T 0x783d6b4
0xff14994 N 0xd1f74
0xb7e460 N 0x6f63898
0x53816c N 0x8779590
0xb7e460 N 0x6f63898
0x9eaf0 N 0x783d6b4
0x53816c T 0x8779590
0xb7e460 N 0x6f63898
0xf37f8 T 0xa89f434
0xe24f7898 N 0x8e39c068
0xb7e460 T 0x6f63898
0xb7e460 N 0x6f63898
0x95666124 T 0x2e588c
0xd374 T 0xf890
0xa40 N 0x57899e8
0xd374 T 0xf890
0xd374 N 0xf890
0xb7e460 N 0x6f63898
0xf37f8 T 0xa89f434
0xff14994 T 0xd1f74
0x95666124 N 0x2e588c
0x53816c N 0x8779590
0x53816c T 0x8779590
0x9eaf0 T 0x783d6b4
0xe24f7898 N 0x8e39c068
0xe24f7898 T 0x8e39c068
0xb7e460 N 0x6f63898
0xe24f7898 N 0x8e39c068
0x9eaf0 N 0x783d6b4
0x53816c N 0x8779590
0xff14994 T 0xd1f74
0x53816c N 0x8779590
0xb7e460 T 0x6f63898
0xb7e460 N 0x6f63898
0x95666124 T 0x2e588c
0xff14994 T 0xd1f74
0xff14994 T 0xd1f74
0xe24f7898 N 0x8e39c068
0xa40 N 0x57899e8
0xf37f8 T 0xa89f434
0xe24f7898 T 0x8e39c068
0x53816c N 0x8779590
0xe24f7898 T 0x8e39c068
0xf37f8 N 0xa89f434
0x95666124 T 0x2e588c
0x9eaf0 T 0x783d6b4
0x9eaf0 T 0x783d6b4
0xf37f8 T 0xa89f434
0x2d0 N 0x0618
0xff14994 T 0xd1f74
0xff14994 N 0xd1f74
0xf37f8 T 0xa89f434
0xe24f7898 N 0x8e39c068
0x53816c N 0x8779590
0x95666124 T 0x2e588c
0xd374 N 0xf890
0xe24f7898 T 0x8e39c068
0x9eaf0 T 0x783d6b4
0xf37f8 T 0xa89f434
0xff14994 N 0xd1f74
0xd374 T 0xf890
0xff14994 N 0xd1f74
0xd374 T 0xf890
0x9eaf0 T 0x783d6b4
0xf37f8 N 0xa89f434
0xff14994 T 0xd1f74
0xb7e460 N 0x6f63898
0xff14994 N 0xd1f74
0xa40 T 0x57899e8
0x2d0 T 0x0618
0xb7e460 N 0x6f63898
0xa40 T 0x57899e8
0xa40 N 0x57899e8
0xd374 N 0xf890
0xf37f8 N 0xa89f434
0x9eaf0 T 0x783d6b4
0xf37f8 T 0xa89f434
0xd374 T 0xf890
0xa40 N 0x57899e8
0xff14994 N 0xd1f74
0xb7e460 T 0x6f63898
0x2d0 T 0x0618
0x2d0 N 0x0618
0xe24f7898 N 0x8e39c068
0x95666124 T 0x2e588c
0xff14994 T 0xd1f74
0xa40 N 0x57899e8
0x2d0 T 0x0618
0xf37f8 N 0xa89f434
0xe24f7898 T 0x8e39c068
0x95666124 T 0x2e588c
0x2d0 N 0x0618
0xd374 N 0xf890
0x2d0 T 0x0618
0xd374 T 0xf890
0x9eaf0 T 0x783d6b4
0xb7e460 N 0x6f63898
0xb7e460 T 0x6f63898
0xff14994 N 0xd1f74
0x53816c T 0x8779590
0xff14994 N 0xd1f74
0x9eaf0 N 0x783d6b4
0xff14994 T 0xd1f74
0xa40 N 0x57899e8
0xff14994 N 0xd1f74
0x2d0 T 0x0618
0xf37f8 T 0xa89f434
0x9eaf0 N 0x783d6b4
0x9eaf0 T 0x783d6b4
0xa40 N 0x57899e8
0x2d0 N 0x0618
0x9eaf0 T 0x783d6b4
0xa40 N 0x57899e8
0x95666124 N 0x2e588c
0x9eaf0 N 0x783d6b4
0xe24f7898 T 0x8e39c068
0x9eaf0 T 0x783d6b4
0xf37f8 T 0xa89f434