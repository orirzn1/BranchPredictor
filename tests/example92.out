0xd374 N 0xd378
0xe24f7898 N 0xe24f789c
0xa40 N 0xa44
0xe24f7898 N 0xe24f789c
0xb7e460 N 0xb7e464
0xa40 N 0xa44
0xff14994 N 0xff14998
0x2d0 N 0x2d4
0x95666124 N 0x95666128
0xf37f8 N 0xf37fc
0x95666124 N 0x95666128
0x95666124 N 0x95666128
0x2d0 T 0x618
0xb7e460 T 0x6f63898
0x95666124 N 0x95666128
0x95666124 N 0x95666128
0xd374 N 0xd378
0xb7e460 N 0xb7e464
0xa40 N 0xa44
0xf37f8 N 0xf37fc
0x95666124 T 0x2e588c
0xb7e460 N 0xb7e464
0x53816c N 0x538170
0x9eaf0 N 0x9eaf4
0xf37f8 N 0xf37fc
0xb7e460 N 0xb7e464
0x95666124 N 0x95666128
0xff14994 T 0xd1f74
0xd374 T 0xf890
0x2d0 T 0x618
0xf37f8 T 0xa89f434
0x95666124 N 0x95666128
0xf37f8 N 0xf37fc
0xa40 N 0xa44
0xf37f8 T 0xa89f434
0x95666124 T 0x2e588c
0xd374 N 0xd378
0x9eaf0 T 0x783d6b4
0xff14994 T 0xd1f74
0xb7e460 N 0xb7e464
0x53816c T 0x8779590
0xb7e460 T 0x6f63898
0x9eaf0 N 0x9eaf4
0x53816c N 0x538170
0xb7e460 N 0xb7e464
0xf37f8 N 0xf37fc
0xe24f7898 N 0xe24f789c
0xb7e460 N 0xb7e464
0xb7e460 N 0xb7e464
0x95666124 T 0x2e588c
0xd374 N 0xd378
0xa40 T 0x57899e8
0xd374 T 0xf890
0xd374 N 0xd378
0xb7e460 T 0x6f63898
0xf37f8 N 0xf37fc
0xff14994 N 0xff14998
0x95666124 N 0x95666128
0x53816c T 0x8779590
0x53816c T 0x8779590
0x9eaf0 N 0x9eaf4
0xe24f7898 N 0xe24f789c
0xe24f7898 N 0xe24f789c
0xb7e460 T 0x6f63898
0xe24f7898 T 0x8e39c068
0x9eaf0 T 0x783d6b4
0x53816c T 0x8779590
0xff14994 N 0xff14998
0x53816c N 0x538170
0xb7e460 N 0xb7e464
0xb7e460 N 0xb7e464
0x95666124 T 0x2e588c
0xff14994 N 0xff14998
0xff14994 N 0xff14998
0xe24f7898 N 0xe24f789c
0xa40 T 0x57899e8
0xf37f8 T 0xa89f434
0xe24f7898 N 0xe24f789c
0x53816c N 0x538170
0xe24f7898 T 0x8e39c068
0xf37f8 T 0xa89f434
0x95666124 T 0x2e588c
0x9eaf0 N 0x9eaf4
0x9eaf0 N 0x9eaf4
0xf37f8 N 0xf37fc
0x2d0 T 0x618
0xff14994 T 0xd1f74
0xff14994 T 0xd1f74
0xf37f8 T 0xa89f434
0xe24f7898 N 0xe24f789c
0x53816c T 0x8779590
0x95666124 T 0x2e588c
0xd374 N 0xd378
0xe24f7898 T 0x8e39c068
0x9eaf0 N 0x9eaf4
0xf37f8 N 0xf37fc
0xff14994 T 0xd1f74
0xd374 T 0xf890
0xff14994 N 0xff14998
0xd374 T 0xf890
0x9eaf0 N 0x9eaf4
0xf37f8 N 0xf37fc
0xff14994 T 0xd1f74
0xb7e460 N 0xb7e464
0xff14994 T 0xd1f74
0xa40 T 0x57899e8
0x2d0 N 0x2d4
0xb7e460 N 0xb7e464
0xa40 T 0x57899e8
0xa40 N 0xa44
0xd374 T 0xf890
0xf37f8 T 0xa89f434
0x9eaf0 T 0x783d6b4
0xf37f8 N 0xf37fc
0xd374 N 0xd378
0xa40 N 0xa44
0xff14994 T 0xd1f74
0xb7e460 T 0x6f63898
0x2d0 N 0x2d4
0x2d0 N 0x2d4
0xe24f7898 N 0xe24f789c
0x95666124 T 0x2e588c
0xff14994 T 0xd1f74
0xa40 N 0xa44
0x2d0 N 0x2d4
0xf37f8 T 0xa89f434
0xe24f7898 N 0xe24f789c
0x95666124 T 0x2e588c
0x2d0 N 0x2d4
0xd374 T 0xf890
0x2d0 T 0x618
0xd374 T 0xf890
0x9eaf0 N 0x9eaf4
0xb7e460 N 0xb7e464
0xb7e460 N 0xb7e464
0xff14994 T 0xd1f74
0x53816c T 0x8779590
0xff14994 T 0xd1f74
0x9eaf0 T 0x783d6b4
0xff14994 T 0xd1f74
0xa40 N 0xa44
0xff14994 T 0xd1f74
0x2d0 T 0x618
0xf37f8 N 0xf37fc
0x9eaf0 N 0x9eaf4
0x9eaf0 N 0x9eaf4
0xa40 N 0xa44
0x2d0 T 0x618
0x9eaf0 T 0x783d6b4
0xa40 N 0xa44
0x95666124 N 0x95666128
0x9eaf0 T 0x783d6b4
0xe24f7898 T 0x8e39c068
0x9eaf0 N 0x9eaf4
0xf37f8 N 0xf37fc
flush_num: 80, br_num: 155, size: 682b
